ip2location.c

```diff
@@ -408,7 +432,7 @@ PHP_FUNCTION(ip2location_get_timezone)
 /* }}} */
 
 
-#if API_VERSION_NUMERIC >= 80100
+#if API_VERSION_NUMERIC == 80100
```

Comment:
I also change this condition, as this not needed anymore with more recent version, when will be released